.github/workflows/dependencies_hip.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+#
+# Copyright 2020-2022 The AMReX Community
+#
+# License: BSD-3-Clause-LBNL
+# Authors: Axel Huebl
+
+set -eu -o pipefail
+
+
+# Ref.: https://rocmdocs.amd.com/en/latest/Installation_Guide/Installation-Guide.html#ubuntu
+curl -O https://repo.radeon.com/rocm/rocm.gpg.key
+sudo apt-key add rocm.gpg.key
+echo 'deb [arch=amd64] https://repo.radeon.com/rocm/apt/debian/ ubuntu main' \
+  | sudo tee /etc/apt/sources.list.d/rocm.list
+echo 'export PATH=/opt/rocm/llvm/bin:/opt/rocm/bin:/opt/rocm/profiler/bin:/opt/rocm/opencl/bin:$PATH' \
+  | sudo tee -a /etc/profile.d/rocm.sh
+
+# we should not need to export HIP_PATH=/opt/rocm/hip with those installs
+
+sudo apt-get update
+
+# Ref.: https://rocmdocs.amd.com/en/latest/Installation_Guide/Installation-Guide.html#installing-development-packages-for-cross-compilation
+# meta-package: rocm-dkms
+# OpenCL: rocm-opencl
+# other: rocm-dev rocm-utils
+sudo apt-get install -y --no-install-recommends \
+    build-essential \
+    gfortran        \
+    libnuma-dev     \
+    libopenmpi-dev  \
+    openmpi-bin     \
+    rocm-dev        \
+    roctracer-dev   \
+    rocprofiler-dev \
+    rocrand-dev     \
+    rocprim-dev
+
+# activate
+#
+source /etc/profile.d/rocm.sh
```

Comment:
What is the purpose of this configuration file `rocm.sh` is the driver of the GPU?